types/flow-typed/recompose_v0.24.x/flow_v0.55.x-/recompose_v0.24.x.js

```diff
@@ -64,64 +64,6 @@ declare module 'recompose' {
 
   declare type UnaryFn<A, R> = (a: A) => R
 
-  declare type Compose = (<A, B, C, D, E, F, G, H, I>(
```

Comment:
Ha, nice to see this go, we hit the limit again not long after I submitted the earlier PR to extend it 😅